packages/web/src/features/register-club/components/ClubButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import styled from "styled-components";
+
+import Card from "@sparcs-clubs/web/common/components/Card";
+import Typography from "@sparcs-clubs/web/common/components/Typography";
+import shadow from "@sparcs-clubs/web/styles/themes/shadow";
+
+interface ClubButtonProps {
+  title: string;
+  buttonText?: string;
```

Comment:
이것두 title처럼 필수로 받으면 좋을 것 같아!

```suggestion
  buttonText: string;
```